runtime/src/lib.rs

```diff
@@ -523,13 +574,25 @@ construct_runtime!(
         Aura: pallet_aura = 23,
         AuraExt: cumulus_pallet_aura_ext = 24,
 
+        // Scheduler
+        Scheduler: pallet_scheduler = 28,
+
+        // Preimage
+        Preimage: pallet_preimage = 29,
+
         // XCM helpers.
         XcmpQueue: cumulus_pallet_xcmp_queue = 30,
         PolkadotXcm: pallet_xcm = 31,
         CumulusXcm: cumulus_pallet_xcm = 32,
         MessageQueue: pallet_message_queue = 33,
 
+        // Contracts
         Contracts: pallet_contracts = 40,
+
+        // Nfts
```

Comment:
Would prefer `// Assets` in line with the assets config module

Reply:
Ah, changed this one to `// Assets` and reverted this morning, forgot to include it back :ok_hand: